langchain_opendataloader_pdf/document_loaders.py

```
@@ -61,12 +62,14 @@ def __init__(
         sanitize: bool = False,
         pages: Optional[str] = None,
         include_header_footer: bool = False,
-        split_pages: bool = True,
+        detect_strikethrough: bool = False,
         hybrid: Optional[str] = None,
         hybrid_mode: Optional[str] = None,
         hybrid_url: Optional[str] = None,
         hybrid_timeout: Optional[str] = None,
         hybrid_fallback: bool = False,
```

Ruff (0.15.7)

[warning] Boolean-typed positional argument in function definition

(FBT001)